data/Application-Dark.css

```diff
@@ -0,0 +1,167 @@
+.roundy-label,
+.roundy-label:hover,
+.roundy-label:selected,
+.roundy-label:selected:focus,
+.roundy-label:hover:selected {
+    background-image: none;
+    background-color: @BLACK_300;
+    box-shadow: none;
+    color: alpha(@text_color, 0.7);
+    font-weight: 700;
+    border-radius: 10px;
+    padding: 0 6px;
+    margin: 0 3px;
+    border-width: 0;
+}
+
+.green {
+    background-color: alpha(@LIME_300, 0.6);
+}
+
+.blue {
+    background-color: alpha(@BLUEBERRY_300, 0.6);
+}
+
+/* .vertical-label,
+.vertical-label:hover,
+.vertical-label:selected,
+.vertical-label:selected:focus,
+.vertical-label:hover:selected {
+
+} */
+
+.label-vertical {
+
```

Comment:
Is something meant to appear here in the future? If not, you might want to remove it to avoid confusion for a reader.